### ISSUE TITLE
--buffer-size feature was lost during a merge

### ISSUE THREAD

**Issue by Foepe** [webfactory/slimdump#64]
In dumptask.php function dump() a new instance of Dumper is created with output but without the buffersize variable. This causes an issue with larger queries and timeouts.

**Comment by mpdude** [comment]
@janopae While you prepated [these changes](https://github.com/webfactory/slimdump/compare/f227b133cea2296e4399a4307b39be2f60ef0027..fe193d0b6ecf4a5fe312e0d08019a694b30d421f), a [new feature and fixes](https://github.com/webfactory/slimdump/compare/f227b133cea2296e4399a4307b39be2f60ef0027..141d02bca66c648aa1003face6d0a28ddac29835) were merged in the master branch.

During [the merge](https://github.com/webfactory/slimdump/commit/3aada5cf71cb8c7d2cbe4fd6975bb999abf6cbba), the `bufferSize` switch (and possibly other changes – did not check!) were lost.

Could you please have a look into this?

**Comment by wesleylancel** [comment]
@mpdude @janopae any chance someone could take a look at this? :) Can confirm that passing the option currently doesn't work which makes it impossible to import dump's on environments where the buffer size is low.

**Comment by janopae** [comment]
@wesleylancel @Foepe Sorry for letting you wait for so long. I made the changes that broke the --buffer-size feature so long ago that I didn't understand what actually went wrong during the merge. So I had to go through the changes that introduced the --buffer-size feature line by line and make sure that everything that has been introduced back then is present in the current code, whose structure changed a little bit since then.

As soon as #78 gets merged, the feature shoud be available again.
